Module/nss/zep_use_chair.nss

```diff
@@ -1,20 +0,0 @@
-//::///////////////////////////////////////////////
```

Comment:
Hang on. You don't want to delete it because that will completely remove it from the module. In this case, it was already in the module so you want to add it back with the original code. Just copy/paste this into a new zep_use_chair.nss file and commit/push it up. Should fix the issue.